docs/faq_build.md

```diff
@@ -60,6 +60,8 @@ You can buy a really unique VID:PID here. I don't think you need this for person
 - https://www.obdev.at/products/vusb/license.html
 - https://www.mcselec.com/index.php?page=shop.product_details&flypage=shop.flypage&product_id=92&option=com_phpshop&Itemid=1
 
+Open-source hardware projects can request a unique [USB-IF](https://www.usb.org/) recognized USB VID:PID by submitting a pull request to [pid.codes](https://pid.codes/). Pid.codes will all alwyas use `0x1209` for the VID. A unique PID is selected when submitting a pull request to pid.codes. Using pid.codes for your VID:PID is not required, and is probably not needed for personal use.
```

Comment:
pid.codes is one site that does this, but Microchip offers the same for their controllers, V-USB has a range too, IIRC.  

* pid.codes
* https://microchip.my.site.com/s/article/How-to-use-a-USB-VID-and-PID-from-the-USB-sub-licensing-program
* https://wiki.openmoko.org/wiki/USB_Product_IDs

And I think there is another group out there that has a license that allows for sub-licensing, too

Reply:
Thanks! I've updated the docs with notes about the other vendors you listed. I wasn't able to find anything regarding USB-V - but I'm a novice and may be looking in the wrong place.